src/scss/styles.scss

```diff
@@ -32,17 +32,16 @@ canvas {
 }
 
 .welcome-section {
-  grid-area: welcome;
-  margin-bottom: 10px;
+  @include margin(0px, 0px, 10px, 0px);
 }
 
 .date {
   background-color: $header-font-color;
   border-radius: 3px;
   float: right;
-  font-size: 10px;
-  margin: $margin-5px;
-  padding: 3px 5px;
+  @include font-size(10px);
```

Comment:
Would a variable or mixin be better suited for our different font sizes?  If we use the mixin in we'd still have to manually change each one correct?  Whereas variables would let us adjust all same-size font texts in one line?

I had built the mixin for font-weight but on the line of thinking above wonder if variables would be better for this styling too?

We may want to think about our missing h3 tag in tandem with this question.